src/main/java/com/example/springresttask/repository/EmailRepositoryImpl.java

```diff
@@ -0,0 +1,121 @@
+package com.example.springresttask.repository;
+
+import com.example.springresttask.domain.Email;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@Slf4j
+public class EmailRepositoryImpl implements EmailRepository {
+    
+    @Value("${spring.datasource.url}")
+    private String URL;
+    @Value("${spring.datasource.username}")
+    private String USERNAME;
+    @Value("${spring.datasource.password}")
+    private String PASSWORD;
+
+    private static Connection connection;
+    private ResultSet resultSet;
+
+    @PostConstruct
+    void init() {
+        try {
+            connection = DriverManager.getConnection(URL, USERNAME, PASSWORD);
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+    }
+
+    @Override
+    public List<Email> findAllByPendingEmail() {
+
+        List<Email> emails = new ArrayList<>();
+        try {
+            //todo:can it be shorter?
+            String SQL = "select id,recipient_name,email_subject,email_body,delivery_date," +
+                    "is_sent from Email e where e.is_sent = false";
+            PreparedStatement preparedStatement = connection.prepareStatement(SQL);
```

Comment:
Resources should be released when they are not needed.
`java.sql.Statement#close`
`java.sql.ResultSet#close`